models/gallery.js

```diff
@@ -0,0 +1,14 @@
+var mongoose = require('mongoose');
```

Comment:
this is a schema file, would be better if you save it in a folder called 'database' or 'db' instead of 'models'